src/domains/strategy_solver_heap_tpolyhedra_sympath.cpp

```diff
@@ -9,7 +9,7 @@ Author: Viktor Malik
 /// \file
 /// Strategy solver for heap-tpolyhedra domain using symbolic paths
 
-// #define DEBUG
+#define DEBUG
```

Comment:
spurious change

Reply:
The file gets completely rewritten in one of the following commits, should I fix this particular commit anyway?